terraform/20-app/ecs.service.public-api.tf

```diff
@@ -48,11 +48,11 @@ module "ecs_service_public_api" {
         },
         {
           name  = "POSTGRES_DB"
-          value = local.aurora.app.public_api_replica.db_name
+          value = local.aurora.app.secondary.db_name
```

Comment:
No functional changes here, and below. We're just renaming the variable to be more descriptive of what it actually is i.e. a generic secondary node address & not a secondary dedicated to the public API